containerd-shim/kata/container.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2018 HyperHQ Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
Hmmm, this package is named "containerd-shim/kata/", since this is kata-runtime, I'm wonder if this `kata` is necessary.

Quesion is, in future, will there be any `containerd-shim/gvisor` or `containerd-shim/xxx` ?

Reply:
Ok, I'll remove directory "kata"